app/Thread.php

```diff
@@ -195,6 +195,9 @@ public function subscriptions()
      */
     public function getIsSubscribedToAttribute()
     {
+        if (! auth()->id()) {
+            return false;
+        }
         return $this->subscriptions()
```

Comment:
Add a empty line before this return statment.

Reply:
I added the empty line but it still failing :(